ksql-parser/src/main/java/io/confluent/ksql/parser/json/LogicalSchemaDeserializer.java

```diff
@@ -25,6 +25,11 @@
 import java.io.IOException;
 
 final class LogicalSchemaDeserializer extends JsonDeserializer<LogicalSchema> {
+  final boolean withImplicitColumns;
+
+  LogicalSchemaDeserializer(final boolean withImplicitColumns) {
```

Comment:
Added this because when deserializing ddl schemas we do want the implicit columns.

Reply:
Out of interest, is it just ROWTIME you need?  I'm hoping to completely remove the concept of 'implicit' column soon by: 
1. making ROWTIME a pseudo column
2. explicitly adding ROWTIME in CT/CS statements if a key column is not provided.

Reply:
> explicitly adding ROWTIME in CT/CS statements if a key column is not provided
you mean ROWKEY?